.gitignore

```diff
@@ -3,3 +3,7 @@ data
 **/.db
 .DS_Store
 .vscode
+docker-compose.yml
+api/openapi_server/configs/development.py
+api/openapi_server/configs/staging.py
+app/vite.config.ts
```

Comment:
Why are these 4 items being added to the .gitignore?

Reply:
I had problems with the port. I modified the config on my end and ignored the files so I didnt push the changes.

Reply:
Gotcha. Could you revert these changes before we merge? We're still going to want to track changes to these files in the future.

Reply:
Update. removed gitignore.